web/profiles/joinup/config/install/views.view.members_overview.yml

```diff
@@ -122,83 +131,245 @@ display:
+            trim_whitespace: false
+            alt: ''
+            rel: ''
+            link_class: ''
+            prefix: ''
+            suffix: ''
+            target: ''
+            nl2br: false
+            max_length: 0
+            word_boundary: true
+            ellipsis: true
+            more_link: false
+            more_link_text: ''
+            more_link_path: ''
+            strip_tags: false
+            trim: false
+            preserve_tags: ''
+            html: false
+          element_type: ''
+          element_class: ''
+          element_label_type: ''
+          element_label_class: ''
+          element_label_colon: false
+          element_wrapper_type: ''
+          element_wrapper_class: ''
+          element_default_classes: true
+          empty: ''
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: true
+          click_sort_column: value
+          type: user_name
+          settings:
+            link_to_entity: false
+          group_column: value
+          group_columns: {  }
+          group_rows: true
+          delta_limit: 0
+          delta_offset: 0
+          delta_reversed: false
+          delta_first_last: false
+          multi_type: separator
+          separator: ', '
+          field_api_classes: false
+          entity_type: user
+          entity_field: name
+          plugin_id: field
+      filters:
+        combine:
+          id: combine
+          table: views
+          field: combine
+          relationship: none
+          group_type: group
+          admin_label: ''
+          operator: word
+          value: ''
           group: 1
-          exposed: false
+          exposed: true
           expose:
-            operator_id: ''
-            label: ''
+            operator_id: combine_op
+            label: Filter
             description: ''
             use_operator: false
-            operator: ''
-            identifier: ''
+            operator: combine_op
+            identifier: combine
             required: false
             remember: false
             multiple: false
             remember_roles:
               authenticated: authenticated
+              anonymous: '0'
+              moderator: '0'
+              administrator: '0'
+            placeholder: ''
           is_grouped: false
           group_info:
-            label: ''
-            description: ''
-            identifier: ''
+            label: 'Combine fields filter'
+            description: null
+            identifier: combine
             optional: true
             widget: select
             multiple: false
             remember: false
             default_group: All
             default_group_multiple: {  }
-            group_items: {  }
-          entity_type: user
-          entity_field: status
-          plugin_id: boolean
+            group_items:
+              1: {  }
+              2: {  }
+              3: {  }
+          fields:
```

Comment:
I find useful to add here also the E-mail. Of course it should be added first as hidden field.

Reply:
I don't think it is would be a good idea to do so. We do not expose the emails in the interface and we do not allow users to see other user's emails. Allowing them to filter by email could unintentionally give users a tool to verify email addresses of other people. That is why I don't usually set up emails in searches. However, I will include this remark and the above one (the label of the search field) in the acceptance criteria of the ticket for the functional team to verify and provide their opinion.